tasks/update-component-sbom/update-component-sbom.yaml

```diff
@@ -0,0 +1,44 @@
+---
+apiVersion: tekton.dev/v1
+kind: Task
+metadata:
+  name: update-component-sbom
+  labels:
+    app.kubernetes.io/version: "0.1.0"
+  annotations:
+    tekton.dev/pipelines.minVersion: "0.12.1"
+    tekton.dev/tags: release
+spec:
+  description: >-
+    Update component-level SBOM with purls with release-time info.
+  params:
+    - name: dataJsonPath
+      description: Relative path to the JSON data file in the workspace.
+    - name: downloadedSbomPath
+      description: |
+        Path to the directory holding previously downloaded SBOMs to be updated.
+      type: string
+      default: downloaded-sboms
+  workspaces:
+    - name: data
+      description: The workspace where the SBOM files reside.
+  results:
+    - name: sbomPath
+      description: Relative path to the updated component-level SBOM in the data workspace.
+  steps:
+    - name: update-component-sbom-purls
+      image: quay.io/konflux-ci/release-service-utils:221d71a4f6b1a50b36b685aa20d86d7df9de33fc
+      script: |
+        #!/usr/bin/env bash
+        set -eux
+        
+        SBOM_PATH="$(dirname "$(params.dataJsonPath)")/$(params.downloadedSbomPath)"
+        INPUT_PATH="$(workspaces.data.path)/${SBOM_PATH}"
+        
+        #update the SBOM files in place
```

Comment:
Why is this happening in place? The `downloadedSbomPath` result description in tasks/push-rpm-data-to-pyxis/push-rpm-data-to-pyxis.yaml says:

> Path to a directory that SBOM files are downloaded onto. The name is differentiated from created or augmented sboms.

So changing it in place seems to contradict that.

Reply:
Now I see https://github.com/konflux-ci/release-service-catalog/pull/656#discussion_r1841692874 - so yeah, fine to do it in place. But then the description should be changed. And maybe the path name/value could be just sbomPath?

Reply:
I updated the description and the var name in `push-rpm-data-to-pyxis.yaml`, but I still want to keep the dir name `downloaded-sboms`, since it still needs to be separate from the created product sboms.

Reply:
It looks good.

BTW, I would like to ask you to stop resolving threads that you didn't create. It's not a hard rule, it's ok for typos for example, but when there is a discussion, the reviewer may want to check if he's happy with the fix of the issue and when somebody else resolves the thread it makes it really hard to keep track of what needs checking.

Reply:
Will do, sorry about that. I usually do that to indicate that the comment is addressed.